index.html

```diff
@@ -2290,10 +2290,14 @@ <h2>Capability Delegation</h2>
         </dl>
 
         <p>
-An example of when this property is useful is when a <a>DID Subject</a>
-chooses to grant their capability to start a vehicle through the combined
-usage of a <a>verification method</a> and the <code>StartCar</code> capability
-to a party other than themselves.
+An example of when this property is useful is when a <a>DID subject</a> chooses
+to delegate their capability to access a protected HTTP API to a party other
+than themselves. In order to delegate the capability, the <a>DID subject</a>
+would use a <a>verification method</a> associated with the
+<code>capabilityDelegation</code> <a>verification relationship</a> to
+cryptographically sign the capability over to another <a>DID subject</a>. The
+delegate would then invoke the capability in a manner that is similar to the
```

Comment:
I know "invoke" is the formal term being used to describe exercising a capability but does "use" make this sentence more inclusive to those with less knowledge?

```suggestion
delegate would then use the capability in a manner that is similar to the
```